<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gethit_exp</class>
 <widget class="QWidget" name="gethit_exp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_levelup">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等级提升！</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProBar_Exp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
